### ISSUE TITLE
C-w hjkl does not update golden-ratio-mode

### ISSUE THREAD

**Issue by BrianHicks** [syl20bnr/spacemacs#497]
With `golden-ratio-mode` on, `C-w` + hjkl does not update the size of the windows until some other event (like a buffer write) takes place.

**Comment by cpaulik** [comment]
This can be enabled in your .spacemacs:

```
(setq golden-ratio-extra-commands
      (append golden-ratio-extra-commands
              '(evil-window-left
                evil-window-right
                evil-window-up
                evil-window-down
                select-window-1
                select-window-2
                select-window-3
                select-window-4
                select-window-5)))
```

**Comment by syl20bnr** [comment]
@cpaulik can you submit a PR for the fix, this configuration can be added here: https://github.com/syl20bnr/spacemacs/blob/master/spacemacs/packages.el#L1057 ?

**Comment by syl20bnr** [comment]
Fixed by #566
